frontend/config/webpack.config.prod.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const path = require("path");
```

Comment:
__[ESLint]__ Error - Parsing error: The keyword 'const' is reserved ([link](https://sider.review/gh/repos/108558043/pulls/347#issue-4065187))

<sub>posted by Sider</sub>